src/canvas.class.ts

```diff
@@ -806,7 +806,8 @@ import { FabricObject } from './shapes/object.class';
             lastX: pointer.x,
             lastY: pointer.y,
             theta: degreesToRadians(target.angle),
-            width: target.width * target.scaleX,
+            width: target.width,
+            height: target.height,
```

Comment:
nothing seems to be using these 2
but it makes sense to store them without scaling

Reply:
probably are super old